app/scripts/services/Usernames.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+	function Usernames ($cookies, $location) {
```

Comment:
Typically services will be singular: `Message`, `Room`. And in future apps I'd suggest what John Papa suggests:

`message.service.js`  
`room.service.js`

Because the are then very self-descriptive. It's up to you if you want to change this one to be singular or not.